server/wfnews-persistence/src/main/java/ca/bc/gov/nrs/wfnews/persistence/v1/dto/PublishedIncidentDto.java

```diff
@@ -108,6 +109,7 @@ public PublishedIncidentDto(PublishedIncidentDto dto) {
 		this.longitude = dto.longitude;
 		this.declaredOutDate = dto.declaredOutDate;
 		this.fireCentre = dto.fireCentre;
+		this.fireCentreName = dto.fireCentreName;
```

Comment:
You added the attribute here, but not in the resource constructor. Add it to the constructor that passes in a PublishedIncident object (line 118)

Reply:
You also didn't add a check in the `equalsAll` method, which is required for equality checks.